my-app/src/components/Body/Taskzone/Zone/Zone.jsx

```diff
@@ -1,18 +1,18 @@
-import "./Task.scss";
+import Card from "./Card/Card";
+import "./Zone.scss";
 
-let Task = (props) => {
+let Zone = (props) => {
+  console.log(props);
   return (
     <div className="task">
       <div className="task-header">
-        <div className="task-name"></div>
+        <div className="task-name">{props.zoneName}</div>
         <div className="task-menu">...</div>
       </div>
       <div className="task-body">
-        <div className="task-card">card 1</div>
-        <div className="task-card">card 2</div>
-        <div className="task-card">card 3</div>
-        <div className="task-card">card 4</div>
-        <div className="task-card">card 5</div>
+        {props.cards.map((card) => (
+          <Card text={card.text} />
+        ))}
       </div>
       <div className="task-footer">
         <div className="add-card"> + Add card</div>
```

Comment:
it would be amazing to add that in future https://aglowiditsolutions.com/blog/react-internationalization-localization/

it is localization and internationalization, pretty widely used, when you need to support multiple countries/languages. I would recommend to add it as a ticket for future to Trello ^